### ISSUE TITLE
Mermaid Diagram codeblock causes error "Top-level await promise never resolved     await mainRunner..."

### ISSUE THREAD

**Issue by cmadland** [quarto-dev/quarto#524]
````qmd
---
title: mermaid-demo
authors:
  - name: Norah Jones
    affiliation: The University
    roles: writing
    corresponding: true
bibliography: references.bib
---

## Section
This is a simple placeholder for the manuscript's main document [@knuth84].


```{mermaid}
flowchart TD
        A["Technology-Integrated Assessment"]
 subgraph subgraph_2rvu6wpa7["Postulated Factors"]
        B("Assessment Purpose")
        C("Duty of Care")
        D("Technology Acceptance")
        E("Assessment Design")
  end
 subgraph subgraph_p3azrijwm["Indicators"]
        F(["Assessment of Learning"])
        G(["Assessment for Learning"])
        H(["Assessment as Learning"])
        I(["Bias"])
        J(["Inclusion"])
        K(["Relationships"])
        L(["Ethical EdTech"])
        M(["Performance Expectancy"])
        N(["Effort Expectancy"])
        O(["Social Influence"])
        P(["Facilitating Conditions"])
        Q(["Measurement Theory"])
        R(["Academic Integrity"])
        S(["Relevance"])
        T(["Reciprocity"])
  end
    A --- B & C & D & E
    B --- F
    F --- G
    G --- H
    C --- I
    I --- J
    J --- K
    K --- L
    D --- M
    M --- N
    N --- O
    O --- P
    E --- Q
    Q --- R
    R --- S
    S --- T
     B:::purpose
     C:::duty
     D:::accept
     E:::design
     F:::purpose
     G:::purpose
     H:::purpose
     I:::duty
     J:::duty
     K:::duty
     L:::duty
     M:::accept
     N:::accept
     O:::accept
     P:::accept
     Q:::design
     R:::design
     S:::design
     T:::design
    classDef purpose fill:#440154,color:#fff,stroke:#440154,stroke-width:4px
    classDef duty fill:#3B528B,color:#fff,stroke:#3B528B,stroke-width:4px
    classDef accept fill:#21918c,color:#fff,stroke:#21918c,stroke-width:4px
    classDef design fill:#5EC962,color:#000,stroke:#5EC962,stroke-width:4px
    style subgraph_2rvu6wpa7 fill:#BBDEFB
    style subgraph_p3azrijwm fill:#C8E6C9
```
````

## Description

When a mermaid codeblock is included in qmd file, the render fails with the following message:

```
Rendering qmd embeds [index.qmd]
Rendering output notebook [index.qmd]
Rendering HTML preview [index.qmd]
pandoc 
  to: html
  output-file: /Users/colin.madland/Documents/GitHub/mermaid-demo/index.html
  standalone: true
  section-divs: true
  html-math-method: mathjax
  wrap: none
  default-image-extension: png
  toc: true
  
metadata
  
  document-css: false
  link-citations: true
  date-format: long
  lang: en
  manuscript:
    article: index.qmd
    notebooks:
      - notebook: index.qmd
        title: Article Notebook
        order: 9999
    mecaFile: index-meca.zip
  notebook-preview-options:
    back: true
  theme: cosmo
  title-block-style: manuscript
  lightbox: auto
  comments:
    hypothesis: true
  title: mermaid-demo
  authors:
    - name: Norah Jones
      affiliation: The University
      roles: writing
      corresponding: true
  bibliography:
    - references.bib
  google-scholar: true
  toc-location: left
  clear-hidden-classes: none
  remove-hidden: all
  unroll-markdown-cells: true
  
error: Top-level await promise never resolved
    await mainRunner(async (args)=>{
    ^
    at <anonymous> (file:///Applications/quarto/bin/quarto.js:118235:5)
```

## Steps to reproduce

- Clone repo from https://github.com/cmadland/mermaid-demo
- render in VSCode

## Expected behaviour

- Project should render html and docx outputs

## Actual behaviour

```
error: Top-level await promise never resolved
    await mainRunner(async (args)=>{
    ^
    at <anonymous> (file:///Applications/quarto/bin/quarto.js:118235:5)
```

## Environment

- VSCode
- Version: 1.92.2 (Universal)
Commit: fee1edb8d6d72a0ddff41e5f71a671c23ed924b9
Date: 2024-08-14T17:29:30.058Z
Electron: 30.1.2
ElectronBuildId: 9870757
Chromium: 124.0.6367.243
Node.js: 20.14.0
V8: 12.4.254.20-electron.0
OS: Darwin arm64 23.6.0

## Quarto Check

```Quarto 1.5.56
[✓] Checking versions of quarto binary dependencies...
      Pandoc version 3.2.0: OK
      Dart Sass version 1.70.0: OK
      Deno version 1.41.0: OK
      Typst version 0.11.0: OK
[✓] Checking versions of quarto dependencies......OK
[✓] Checking Quarto installation......OK
      Version: 1.5.56
      Path: /Applications/quarto/bin

[✓] Checking tools....................OK
      TinyTeX: (external install)
      Chromium: (not installed)

[✓] Checking LaTeX....................OK
      Using: TinyTex
      Path: /Users/colin.madland/Library/TinyTeX/bin/universal-darwin
      Version: 2024

[✓] Checking basic markdown render....OK

[✓] Checking Python 3 installation....OK
      Version: 3.12.3
      Path: /usr/local/bin/python3
      Jupyter: 5.7.2
      Kernels: python3

[✓] Checking Jupyter engine render....OK

[✓] Checking R installation...........OK
      Version: 4.3.3
      Path: /Library/Frameworks/R.framework/Resources
      LibPaths:
        - /Library/Frameworks/R.framework/Versions/4.3-arm64/Resources/library
      knitr: 1.45
      rmarkdown: 2.27

[✓] Checking Knitr engine render......OK
```

**Comment by cscheid** [comment]
This is caused by a recent change to Google Chrome. We've fixed the problem in the 1.5 branch, a new version is going to be released on Sep 1st. In the meantime, you can install the 1.6 prereleases that also have the fix.

**Comment by cmadland** [comment]
confirming that installing Quarto 1.6.8  pre-release resolves the issue.
